calebwagner/react-notes-app
# Cjw edit note

#### Changes Made
1. Modified file `NotebookEdit.js` and `NoteEdit.js` to include edit functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-edit-note
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > in browser, create a notebook and you should see it at the bottom of the page.
    > do the same with notes and once note is saved the new note will solely appear.
4. View code file.
    > Confirm file modifications are present as indicated above.
